dvc/command/pkg.py

```diff
@@ -4,27 +4,92 @@
 import logging
 
 from dvc.exceptions import DvcException
-from dvc.command.base import CmdBase, fix_subparsers, append_doc_link
+
+from .config import CmdConfig
+from .base import CmdBase, fix_subparsers, append_doc_link
+
+from dvc.pkg.config import pkg_add, pkg_remove, pkg_modify, pkg_list
```

Comment:
It still feels that config is exposed too much (the same for remote). The only thing commands should see is probably repo object - they should not care if it goes into config or a separate or database, or whatnot. Like all these commands `pkg_*` and `remote_*` should be part of the Repo? Of course w/o creating an unmanageable class out of Repo. Some of the functionality can go to the Remote and/or Pkg class - like modify, dump to config, etc.

Reply:
Config can exist outside of repo(e.g. `--system` and `--global`, where `repo` doesn't make much sense), so it is a more generic object. Maybe we could create something like RemoteConfig/PkgConfig with those methods. Created https://github.com/iterative/dvc/issues/2080 to revisit it later.

Reply:
good point with global and system. Will it be possible to run `dvc pkg add --global` or `pkg list` outside of a repo?

Reply:
Yes.